src/v3/pages/reports/meeting_attendance/components/report_toolbar/index.types.tsx

```diff
@@ -0,0 +1,11 @@
+import { TFunction } from 'i18next';
+
+export type MeetingAttendanceReportToolbarProps = {
+  t: TFunction<'translation', undefined>;
```

Comment:
@Logan08691: this should not be needed since the `t` function from our hook can already be used in code. Thank you.